cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -273,6 +273,23 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     )
   }
 
+  @Test def showConstraints() {
+    executePreparationQueries {
+      List("CREATE CONSTRAINT name ON (c:Country) ASSERT c.name is UNIQUE")
+    }
+
+    profileQuery(
+      title = "List constraints",
```

Comment:
For consistency I would prefer the title to be "Show constraints" (remember to also update the link in the execution plans table)